plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/StatisticsNavigatorNodeRenderer.java

```diff
@@ -192,6 +200,17 @@ private void drawObjectDescription(@NotNull GC gc, @NotNull DBNDatabaseNode node
         }
     }
 
+    private void drawObjectChildrenCounter(@NotNull GC gc, @NotNull DBNDatabaseNode node, @NotNull Rectangle bounds) {
+        int childCount = 0;
+        try {
+            childCount = node.getChildren(new VoidProgressMonitor()).length;
+        } catch (DBException e) {
+            log.error("Error fetching children for node: " + node.getNodeDisplayName());
```

Comment:
Logging could be omitted - otherwise, in the case of an error, log files will be flooded with the same errors. Moreover, you don't log the exception, so its value is worthless.